runtime/queries/metricsview_search.go

```diff
@@ -68,27 +64,10 @@ func (q *MetricsViewSearch) Resolve(ctx context.Context, rt *runtime.Runtime, in
 		return err
 	}
 
-	for _, d := range q.Dimensions {
-		if !sec.CanAccessField(d) {
-			return ErrForbidden
-		}
-	}
-
-	olap, release, err := rt.OLAP(ctx, instanceID, mv.Connector)
-	if err != nil {
-		return err
-	}
-	defer release()
-
-	if olap.Dialect() != drivers.DialectDuckDB && olap.Dialect() != drivers.DialectDruid && olap.Dialect() != drivers.DialectClickHouse && olap.Dialect() != drivers.DialectPinot {
-		return fmt.Errorf("not available for dialect '%s'", olap.Dialect())
-	}
-
-	if mv.TimeDimension == "" && !isTimeRangeNil(q.TimeRange) {
-		return fmt.Errorf("metrics view '%s' does not have a time dimension", mv)
-	}
-
 	if !isTimeRangeNil(q.TimeRange) {
+		if mv.TimeDimension == "" {
+			return fmt.Errorf("metrics view '%s' does not have a time dimension", mv)
+		}
 		start, end, err := ResolveTimeRange(q.TimeRange, mv)
```

Comment:
Can this be removed now? I think the `rewriteQueryTimeRanges` rewriter should resolve time ranges